db/schema.rb

```diff
@@ -506,11 +506,6 @@
     t.index ["community_user_id"], name: "index_warnings_on_community_user_id"
   end
 
-  add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
-  add_foreign_key "audit_logs", "communities"
-  add_foreign_key "audit_logs", "users"
-  add_foreign_key "categories", "licenses"
-  add_foreign_key "categories", "tag_sets"
```

Comment:
Why the removal of these lines?

Reply:
Huh, that's strange, I didn't intentionally remove them, I don't even remember opening this file. I can add them back in.

Reply:
okay added back! I must have opened the file and hit backspace over a selected section without realizing it. /bonk.